Sources/Afluent/SequenceOperators/OutputSequence.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  OutputSequence.swift
```

Comment:
Nitpick: Should we call this file `OutputAtSequence`?

Reply:
I plan to add `output(in:)` here as well. Do you think they should be in different files?